src/components/KnownFollowers.tsx

```diff
@@ -164,29 +165,22 @@ function KnownFollowersInner({
               },
             ]}
             numberOfLines={2}>
-            {count > 2 ? (
+            {renderableCount >= 2 ? (
               <Trans>
-                Followed by{' '}
+                {_(msg`Followed by`)}{' '}
```

Comment:
This kind of negates the point of using `Trans` above. The point was to determine the format first ("Followed by X", "Followed by X and Y" or "Followed by X, Y and Z" first), and then interpolate into one of the supported formats. Breaking `Followed by` into its own `msg` breaks that.

Reply:
See https://github.com/bluesky-social/social-app/pull/4494 for context where this was introduced.

Reply:
Oh ok, TIL, yeah can fix this up 👍

Reply:
2-for-1, realized when splitting out this logic that I was missing a case:
![CleanShot 2024-06-18 at 15 42 46@2x](https://github.com/bluesky-social/social-app/assets/4732330/27ef77b2-10b5-484a-98c6-e3571cfb4534)